.github/workflows/cd.yaml

```
@@ -0,0 +1,127 @@
+          builder: ${{ steps.buildx.outputs.name }}
+          tags: lanelet2_src
+          load: true
+          target: lanelet2_src
+
+      - name: Create output directory
+        run: mkdir -p ${{ github.workspace }}/output
+
+      - name: Do stuff inside the container
+        uses: addnab/docker-run-action@v3
+        with:
+          image: lanelet2_src
+          shell: bash
+          options: -v ${{ github.workspace }}/dist:/dist
+          run: |
+            set -e
+            export WORKSPACE=$(pwd)
+
+            # find python version
+            export PYTHON_VERSION=$(python3 -c "import sys; print('.'.join([str(i) for i in sys.version_info[:2]]))")
+
+            # build lanelet2 with python
+            source /opt/ros/noetic/setup.bash
+            catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release -DPYTHON_VERSION=$PYTHON_VERSION
+            catkin build --no-status
+
+            # assemble python package contents
+            cp -r $WORKSPACE/devel/.private/lanelet2_python/lib/python3/dist-packages/lanelet2 $WORKSPACE/lanelet2
+            export LANELET2_VERSION=$(catkin_package_version $WORKSPACE/src/lanelet2)
+            sed 's/{{ version }}/'"$LANELET2_VERSION"'/' $WORKSPACE/src/lanelet2/lanelet2_python/setup.py.template > $WORKSPACE/setup.py
+          
+            # install patchelf
+            cd $HOME
+            wget https://github.com/NixOS/patchelf/releases/download/0.17.2/patchelf-0.17.2-x86_64.tar.gz
+            tar -xzf patchelf-0.17.2-x86_64.tar.gz
+            sudo ln -s $HOME/bin/patchelf /bin/patchelf
+            patchelf --version
+
+            # install pip
+            export PATH="$PATH:$HOME/.local/bin"
+            cd $HOME
+            wget -q https://bootstrap.pypa.io/get-pip.py
+            python3 get-pip.py
+            pip install auditwheel
+
+            # repair wheel
+            cd $WORKSPACE
+            pip wheel -w broken-dist/ .
+            auditwheel repair -w dist/ --plat manylinux_2_31_x86_64 broken-dist/*.whl
+            
+            # copy wheel to dist directory
+            sudo cp -r $WORKSPACE/dist/* /dist
+
+      - name: Store wheel
+        uses: actions/upload-artifact@v3
+        with:
+          name: dist
+          path: dist/
+
+  test:
+    needs: build
+    strategy:
+      matrix:
+        # test only on currently supported version
+        # python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8"]
+        os: ["ubuntu-22.04", "ubuntu-20.04"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Restore wheel
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+          path: dist/
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Test wheel
+        run: |
+          pip install dist/*
+          python -c "import lanelet2; assert lanelet2.core.Point2d(0, 0, 0) is not None"
```

```suggestion
      - name: Install wheel
        run: |
          pip install dist/*
      - name: Test installation
        run: |
          python -c "import lanelet2; assert lanelet2.core.Point2d(0, 0, 0) is not None"
```
Just an idea, to distinguish failure during installation vs runtime failure